frontend/src/index.tsx

```diff
@@ -32,6 +35,12 @@ const router = createBrowserRouter([
         action: destructionListCreateAction,
         loader: destructionListCreateLoader,
       },
+      {
+        path: "/destruction-lists/:id/review",
```

Comment:
Pleas keep the url structure logical, the path prefix for everything destruction list (currently) is `/destruction-list/` please extend that on every related route. A detail page would be `/destruction-list/<uuid>` and some other page for that item would be `/destruction-list/<uuid>/some-other-page`. In this case we could use `/destruction-list/:id/review` (without the plural)

Motivation: This keeps urls structured and predictable, it also allows manipulating the url as a means for navigation and makes relative urls easier to work with.

Reply:
It currently seems that the prefix is `/destruction-lists` for the "create" page as well. Shall I change both to `destruction-list` then?